waitress/app/utils.py

```diff
@@ -14,6 +14,26 @@ class UserRepository(object):
     A wrapper class for methods that update the list of users.
     """
 
+    @staticmethod
+    def add(user_type, username):
```

Comment:
This was written to accommodate creation of any type of user. For the moment, what we need is just a guest.